README.md

```diff
@@ -13,7 +13,20 @@ DATABASE_URL=Server=localhost;port=5432;Database=WhaleSpottingDb;user id=usernam
 ```
 where the user id and password are your own PSQL credentials (note: the default username is postgres, if you're not sure!). 
 
-To create your own database, run `dotnet ef database update`. This will initialize an empty database.
+To create/update your database, in `WhaleSpottingContext.cs`, comment out the following line:
```

Comment:
I think this was only for creating migrations so when we run 'dotnet ef migrations add ...' rather than for creating or applying migrations which is done with dotnet run where we have access to the environment variable :)